parsons/action_network/action_network.py

```diff
@@ -209,7 +209,7 @@ def get_tag(self, tag_id):
             A  JSON of the entry. If the entry doesn't exist, Action Network returns
             "{'error': 'Couldn't find tag with id = <id>'}"
         """
-        return self.api.get_request(url=f"{self.api_url}/tags/{tag_id}")
+        return self.api.get_request(url=f"tags/{tag_id}")
```

Comment:
Could you do this for all of the methods in the AN class as well?

Reply:
I thought I got all of the methods in the ActionNetwork class. Could you put out an example of what you are seeing needs to be fixed?

Reply:
My mistake. Misread the code.